execution/executor-types/src/lib.rs

```diff
@@ -0,0 +1,348 @@
+// Copyright (c) The Libra Core Contributors
```

Comment:
might be worth splitting this up into individual files, but I'll leave that up to you.

Reply:
The argument here is some structs here will be cleaned up later so I will revisit this after I finish the most refactoring work.